main.cpp

```diff
@@ -658,6 +770,20 @@ void FXSection::displayInitState()
               << "Mix = " << mix << std::endl;
 }
 
+void FXSection::iterateCompRatio(float ratio)
+{
+    if (ratio < compressionRatio)
+    {
+        std::cout << "initial ratio is " << ratio << std::endl;
+        float difference = (compressionRatio - ratio) * 2;
+        for (int i = 0; i < static_cast<int>(difference); ++i)
+            {
+                ratio += 0.25f;
+                std::cout << "current ratio is " << ratio << std::endl;
+            }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above

FIXME: Casting has not been introduced.  Do not use casting.  Figure out another way to eliminate any warnings produced by this code.
- [ ] revise